src/keyring/KeyringController.test.ts

```diff
@@ -40,11 +46,16 @@ describe('KeyringController', () => {
     keyringTypes: string[];
     keyrings: Keyring[];
   };
-  const baseConfig: Partial<KeyringConfig> = { encryptor: new MockEncryptor() };
+  const additionalKeyrings = [QRKeyring];
+  const baseConfig: Partial<KeyringConfig> = {
+    encryptor: new MockEncryptor(),
+    keyringTypes: additionalKeyrings,
+  };
```

Comment:
Mind adding an extra blank line after this line? I think it'll make the code look less smushed.